+global-variations.sk

```diff
@@ -256,3 +256,7 @@ global:
 		jungle [wood] = minecraft:jungle_-
 		acacia [wood] = minecraft:acacia_-
 		dark oak [wood] = minecraft:dark_oak_-
+
+	{nether_block}:
```

Comment:
This seems a little off to me. Part of the complexity is that there are non-wooden blocks that exist with crimson and warped variants, like the Nylium, but if you do it as a separate variation like this rather than including them in the main `{wood type}` variation above then there's a lot more manual work to keep up with rather than them automatically being included with all other wood types.

IMO we should rename this entire `global` section to `wood types before 1.16:` which is `minecraft version = 1.15 or older` and then add a new `wood types after 1.16:` which is `minecraft version = 1.16 or newer`. The 1.16 one would contain this new `{nether_block}` variation (which should be renamed to `{nether block}` for consistency) as well as a new definition for the `{wood type}` variation which includes crimson and warped as wood types. Then most of the wooden stuff (signs, planks, etc.) for crimson/warped wood would come automatically rather than you having to explicitly define them, and for the non-woooden blocks that exist in both variants you'd use the `{nether block}` variation.

Reply:
The reason I didn't add this to wood, was because they didn't add boats for these new "wood" types. They might possibly add boats in a future snapshot, and I was planning on changing it at that time.

Im going to do the changes regarding the section and renamed `{nether_block}` to `{nether block}` just for now. 

As Im typing this, I noticed boats do not use the wood time variation so we might be fine. 
Ill make the changes as requested, and once we get 1.16 and a working version of Skript for it, I/we can do more testing on this.

Reply:
Sounds good! This plan is a good fallback if it proves too complicated in some way to include them as a wood type.

Reply:
so far its working out good. Im shuffling a few tiny things around just to make them a tad more future proof.